index.html

```diff
@@ -200,4 +200,4 @@ <h2 class="seventh-sheet__contact-me">Contact me</h2>
   </section>
   <script src="main.js"></script>
 </body>
```

Comment:
- [ ] [OPTIONAL] It would be nicer if you fixed the mobile version, currently it leaves a blank margin on the right side
<img width="322" alt="image" src="https://user-images.githubusercontent.com/61408860/168894580-608d890e-ce36-4070-9489-1f76d62bf00b.png">